ModPatches/More Armory! Midworld Arms Pack/Patches/More Armory! Midworld Arms Pack/ThingDefs_Apparel.xml

```diff
@@ -0,0 +1,330 @@
+<?xml version="1.0" encoding="utf-8"?>
+<Patch>
+
+	<!-- === Combat Helmet === -->
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="Apparel_MA_CombatHelmet"]/statBases</xpath>
+		<value>
+			<Bulk>7</Bulk>
+			<WornBulk>3</WornBulk>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Apparel_MA_CombatHelmet"]/statBases/ArmorRating_Sharp</xpath>
+		<value>
+			<ArmorRating_Sharp>3.8</ArmorRating_Sharp>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Apparel_MA_CombatHelmet"]/statBases/ArmorRating_Blunt</xpath>
+		<value>
+			<ArmorRating_Blunt>30</ArmorRating_Blunt>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Apparel_MA_CombatHelmet"]/costList/Steel</xpath>
+		<value>
+			<Steel>65</Steel>
+			<DevilstrandCloth>20</DevilstrandCloth>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationRemove">
+		<xpath>Defs/ThingDef[defName="Apparel_MA_CombatHelmet"]/costList/ComponentIndustrial</xpath>
+	</Operation>
+	
+	<!-- === Sentry Helmet === -->
+	<Operation Class="PatchOperationAdd">
+		<xpath>Defs/ThingDef[defName="Apparel_MA_SentryHelmet"]/statBases</xpath>
+		<value>
+			<Bulk>8.5</Bulk>
+			<WornBulk>4.5</WornBulk>
+		</value>
+	</Operation>
+
+	<Operation Class="PatchOperationReplace">
+		<xpath>Defs/ThingDef[defName="Apparel_MA_SentryHelmet"]/statBases/ArmorRating_Sharp</xpath>
+		<value>
+			<ArmorRating_Sharp>4.1</ArmorRating_Sharp>
```

Comment:
The helmet is stuffable and lacks a ArmorRating_Sharp node, so this patch fails.